back/engines/commercial/granular_permissions/spec/acceptance/permissions_spec.rb

```diff
@@ -234,5 +263,107 @@
         })
       end
     end
+
+    get 'web_api/v1/permissions/:action/requirements' do
```

Comment:
Is this the one to be used for global registration? Initiatives is a different endpoint, right?

Reply:
This is the same one as for initiatives. Both are "nested" under the global scope.

Reply:
The endpoint for initiatives was already in place and is consistent with the `participation_conditions` that was already there.

Reply:
All I was thinking was that initiative actions could have different requirements than for the global registration, but then actually we have that additional 'visiting' action that will be used by the global registration and that will allow it to be different from the initiative actions.

Reply:
Indeed, the `visiting` action separates this permission from the initiative permissions, which have their own actions.